utils/log-target.js

```diff
@@ -58,49 +77,19 @@ module.exports = class LoggingTarget {
 
   /** Construct the service bucket name for Stackdriver logging. */
   get serviceName() {
-    // TODO: Drastically reduce combinatoral explosion of buckets.
-    const {
-      singlePassType,
-      referrer,
-      runtime,
-      thirdParty,
-      binaryType,
-      canary,
-      assert,
-      expected,
-    } = this.opts;
-    const name = ['default'];
-
-    if (singlePassType) {
-      name.push(singlePassType);
-    }
-    name.push(CDN_REGEX.test(referrer) ? 'cdn' : 'origin');
-
-    if (runtime) {
-      name.push(runtime);
-    } else if (thirdParty) {
-      name.push('3p');
-    } else {
-      name.push('1p');
-    }
+    const { referrer, version, expected } = this.opts;
+    const rtvPrefix = version.substr(0, 2);
 
-    // Do not append binary type if 'production' since that is the default
-    if (binaryType) {
-      if (binaryType !== 'production') {
-        name.push(binaryType);
-      }
-    } else if (canary) {
-      name.push('canary');
-    }
+    const name = [CDN_REGEX.test(referrer) ? 'CDN' : 'Origin'];
+    name.push(CHANNEL_TYPES[rtvPrefix] || 'Unknown');
 
-    if (assert) {
-      name.push('user');
-    }
-    if (expected) {
-      name.push('expected');
+    if ([logs.ads, logs.users].includes(this.getLog())) {
```

Comment:
Why not append `(Ads)` or `(User)`? I don't think either is really an expected error, so grouping this way seems surprising.

Reply:
Because errors in either case go into a separate reporting project, where every single service would be `(Ads)` or `(User)`, so there's kinda no point.